src/routes/results/[results_string]/+page.svelte

```diff
@@ -74,10 +81,24 @@
 				</ul>
 				<div class="actions">
 					<!-- TODO: add email action -->
-					<button class="btn secondary">Email Your Results</button>
-					<a href="https://www.allwecansave.earth/dynamics-resources" class="btn secondary"
-						>Resources for Support</a
+					<form
+						onsubmit={async (e) => {
+							e.preventDefault();
+							const resp = await _sendEmail(email, data.object);
+							if (resp.success) {
+								alert('Email sent!');
+							} else {
+								alert("Sorry! Something went wrong and we couldn't send the email.");
+							}
+						}}
 					>
+						<input value={email} oninput={handleEmailChange} placeholder="Email address" />
```

Comment:
tip: if you add `type="email"` here that should automatically prevent form submission if the input doesn't match accepted email formats. We should still validate again on the backend, but it's a nice cheap user enhancement!

Reply:
ooh very nice, will update!